wpilib-sys/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# This file is part of "first-rust-competition", which is free software: you
+# can redistribute it and/or modify it under the terms of the GNU General
+# Public License version 3 as published by the Free Software Foundation. See
+# <https://www.gnu.org/licenses/> for a copy.
+
+[package]
+name = "wpilib-sys"
+version = "0.2.2"
+authors = ["Josh Hejna <josh.hejna@gmail.com>"]
+build = "build.rs"
+description = "FRC's WPILib system bindings for rust."
+repository = "https://github.com/Lytigas/first-rust-competition"
+keywords = ["frc", "RoboRIO", "wpilib", "RoboRust", "wpilib-sys"]
+categories = ["embedded", "hardware-support", "external-ffi-bindings"]
+license = "GPL-3.0"
+
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "build.rs",
+    "HAL/WPILIB_LICENSE",
+    "HAL/include/**/*",
+    "HAL/lib/**/*",]
```

Comment:
You'll want to double-check these includes actually match what you're expecting.

Reply:
Yea, I never checked these.